src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -17,7 +17,7 @@ import backendServices from 'platform/user/profile/constants/backendServices';
 import get from 'platform/utilities/data/get';
 import recordEvent from 'platform/monitoring/record-event';
 
-import ProfileFieldHeading from 'applications/personalization/profile360/vet360/components/base/ProfileFieldHeading';
```

Comment:
how do we feel about app specific aliases? I think the guiding principle up until this point is to only alias site wide references (e.g. helpers used in all apps) since this adds the burden of knowing aliases for each app to do maintenance. 

if we go this route do we want to have guidelines for app aliases, document them on the client document site, or can we make it more generic and easier to grok by aliasing all of the applications by aliasing `/applications`?

Reply:
We could also look into letting [each app manage parts](https://survivejs.com/webpack/developing/composing-configuration/) of the webpack config and use webpack's [resolve.alias](https://webpack.js.org/configuration/resolve/#resolvealias) instead of babel's? not sure what the trade offs over the other are.

Reply:
@rianfowler sorry I missed the notification of your comments here.